account-kit/wallet-client/src/client/actions/requestAccount.ts

```diff
@@ -77,9 +75,7 @@ export async function requestAccount(
 
   const account = await createAccount({
     accountAddress,
-    counterfactualInfo: counterfactualInfo
```

Comment:
this was totally unncessary. we just got this data back from the server & it should already match this shape.